src/components/data-management/FileUploadModal.jsx

```diff
@@ -57,31 +55,11 @@ const FileUploadModal = (props) => {
       pushNotificationMessage('error', 'Only files contained in folder are accepted');
     }
 
-    filteredFiles.forEach((file) => {
-      let fileName = null;
-      const error = [];
-      // First character of file.path === '/' means a directory is uploaded
-      // Remove initial slash so that it does not create an empty directory in S3
-      const paths = file.path.split('/');
-      fileName = `${paths[paths.length - 2]}/${paths[paths.length - 1]}`;
-      const valid = checkIfFileValid(fileName, selectedTech);
-
-      if (!valid.isValidType) error.push('Invalid file type.');
-      if (!valid.isValidFilename) error.push('Invalid file name.');
-
-      newList.push({
-        name: fileName,
-        bundle: file,
-        upload: {
-          status: UploadStatus.UPLOADING,
-          progress: 0,
-        },
-        valid: valid.isValidType && valid.isValidFilename,
-        errors: error.join(', '),
-      });
-    });
+    const newFiles = await Promise.all(filteredFiles.map(async (file) => (
```

Comment:
dont think the function passed in map needs to be async

Reply:
Correct, I also noticed this but forgot! Thanks